siliconcompiler/tools/openroad/scripts/sc_write_images.tcl

```diff
@@ -124,6 +166,7 @@ proc sc_image_routing_congestion {} {
     "Routing" \
     "routing_congestion.png" \
     "routing congestion"
+    0
```

Comment:
So we are not using your new logic from the `sc_image_heatmap()` proc here?

Reply:
the routing congestion is a little more limited so we cannot use that logic yet